tbin/basic_test.go

```diff
@@ -6,10 +6,11 @@ package tbin
 import (
 	"bytes"
 	"fmt"
-	"github.com/ardielle/ardielle-go/rdl"
 	"io/ioutil"
 	"os"
 	"testing"
+
+	"github.com/ardielle/ardielle-go/rdl"
```

Comment:
this is goimports convention of separating standard and third-party imports, switching to it would avoid contributors using goimports having to struggle with spurious diffs due to the order of imports :smile: